src/ContinuousTransformations.jl

```diff
@@ -1215,7 +1233,7 @@ type and stucture, and `len` is the length of the result.
 function make_ungrouped_container(::Type{Vector},
                                   representative_elt::AbstractArray{T},
                                   len) where T
-    [Vector{T}(len) for _ in CartesianRange(indices(representative_elt))]
+    [Vector{T}(len) for _ in CartesianIndices(indices(representative_elt))]
```

Comment:
don't we need an `undef` here?

Reply:
Sorry, I don't see how you mean to use `undef` in this  context